applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.cpp

```diff
@@ -15,1258 +15,1473 @@
 #define M_PI 3.1415926535897932384626433832795
 #endif // M_PI
 
-

```

Comment:
I think most of what is marked as being changed is the result of a style formatting tool, sorry for that.
@peterjwilson can you please comment on what has actually changed?